ophyd/sim.py

```diff
@@ -768,9 +744,7 @@ class ABDetector(Device):
     def trigger(self):
         return self.a.trigger() & self.b.trigger()
 
-    @property
-    def hints(self):
-        return {'fields': [self.a.name]}
+    _default_hints = {'fields': ['a']}
```

Comment:
minor style point: can you move this up to the top of the class definition?

Reply:
Sure, done for this case, and for a few other similar cases.